tests/system_tests/test_i23_system.py

```diff
@@ -1,18 +1,26 @@
 import os
 
+from bluesky.run_engine import RunEngine
+
 from dodal.utils import make_all_devices
 
 if __name__ == "__main__":
     """This test runs against the real beamline and confirms that all the devices connect
-    i.e. that we match the beamline PVs. Obviously this must be run on the DLS network
-    and could be flaky if parts of the beamline are down for maintenance.
+    i.e. that we match the beamline PVs.
+
+    THIS TEST IS ONLY EXPECTED TO WORK FROM AN I23 WORKSTATION! This is because i23 uses an
+    EPICS v4 (pva) device, and no PVA gateways currently exist.
```

Comment:
Must: I think a better way of fixing this is that the device should be skipped if not on the right machine